testutils/test_server.go

```diff
@@ -60,15 +60,17 @@ type TestServer struct {
 	// as part of the TestServer (including the server).
 	channelStates map[*tchannel.Channel]*tchannel.RuntimeState
 
-	verifyOpts *goroutines.VerifyOpts
-	postFns    []func()
+	introspectOpts *tchannel.IntrospectionOptions
```

Comment:
Don't think we need the method to modify the options, but it's nice to specify the options we're using in one place. Introducing this field removed 3 or 4 places we were hard-coding the same options.